packages/custom-webpack/src/custom-webpack-builder.ts

```diff
@@ -73,3 +81,22 @@ async function resolveCustomWebpackConfig(
 
   return loadModule(path);
 }
+
+function logConfigProperties(
+  config: CustomWebpackBuilderConfig,
+  webpackConfig: Configuration,
+  logger: logging.LoggerApi
+): void {
+  // There's no reason to log the entire configuration object
+  // since Angular's Webpack configuration is huge by default
+  // and doesn't bring any meaningful context by being printed
+  // entirely. Users can provide a list of properties they want to be logged.
+  if (config.verbose?.properties) {
+    for (const property of config.verbose.properties) {
+      const value = get(webpackConfig, property);
+      if (value) {
+        logger.info(value);
```

Comment:
I'm not sure it will print the whole thing properly. I mean yeah, if the property is a primitive then sure, but what if I want to print out, say, `loaders`? Or `plugins`, especially if one of them is a class instance with a constructor. I don't think that `logger.info` knows how to deal with it. That's the challenge :)

Reply:
Yeah that's true. Lemme just check how their logger prints objects. I'll respond in the chat. We prolly have 3 ways to go with:
- use their logger
- use 3rd-party logger
- serialize the object into a string and log as a string (meh)

Reply:
I think we should stick with their logger, so probably the `meh` option is the way to go 😅.  
Whether to use some 3rd party for serialization or not is another question, but as for logger I believe it would be best to continue using their logger.